docs/index.md

```diff
@@ -156,19 +151,13 @@ Usage: (Options preceded by an asterisk are required) [options]
     --executorForcePullImage
        Option to force pull the executor image. [DOCKER MODE ONLY]
        Default: false
-    --executorHealthDelay
-       The delay between executor healthcheck requests (ms).
-       Default: 30000
     --executorImage
-       The docker executor image to use. [DOCKER MODE ONLY]
-       Default: mesos/elasticsearch-executor
+       The docker executor image to use. E.g. 'elasticsearch:latest' [DOCKER
+       MODE ONLY]
+       Default: elasticsearch:latest
     --executorName
        The name given to the executor task.
        Default: elasticsearch-executor
```

Comment:
Not for now. Maintain backwards compatibility and ensure people know they are executors.